docs/content/快速上手/mongodb-tutorial-zh.md

```diff
@@ -0,0 +1,216 @@
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongouser
+      - MONGO_INITDB_ROOT_PASSWORD=mongopw
+  elasticsearch:
+    image: elastic/elasticsearch:7.6.0
+    environment:
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+  kibana:
+    image: elastic/kibana:7.6.0
+    ports:
+      - "5601:5601"
+```
+
+2. 进入 MongoDB 容器，初始化副本集和数据:
+```
+docker-compose exec mongo /usr/bin/mongo -u mongouser -p mongopw
+```
+
+```javascript
+// 1. 初始化副本集
+rs.initiate();
+rs.status();
+
+// 2. 切换数据库
+use mgdb;
+
+// 3. 初始化数据
+db.orders.insertMany([
+  {
+    order_id: 101,
+    order_date: ISODate("2020-07-30T10:08:22.001Z"),
+    customer_id: 1001,
+    price: NumberDecimal("50.50"),
+    product: {
+      name: 'scooter',
+      description: 'Small 2-wheel scooter'
+    },
+    order_status: false
+  },
+  {
+    order_id: 102, 
+    order_date: ISODate("2020-07-30T10:11:09.001Z"),
+    customer_id: 1002,
+    price: NumberDecimal("15.00"),
+    product: {
+      name: 'car battery',
+      description: '12V car battery'
+    },
+    order_status: false
+  },
+  {
+    order_id: 103,
+    order_date: ISODate("2020-07-30T12:00:30.001Z"),
+    customer_id: 1003,
+    price: NumberDecimal("25.25"),
+    product: {
+      name: 'hammer',
+      description: '16oz carpenter hammer'
+    },
+    order_status: false
+  }
+]);
+
+db.customers.insertMany([
+  { 
+    customer_id: 1001, 
+    name: 'Jark', 
+    address: 'Hangzhou' 
+  },
+  { 
+    customer_id: 1002, 
+    name: 'Sally',
+    address: 'Beijing'
+  },
+  { 
+    customer_id: 1003,
+    name: 'Edward',
+    address: 'Shanghai'
+  }
+]);
+```
+
+3. 下载以下 jar 包到 `<FLINK_HOME>/lib/`:
+
+```下载链接只在已发布的版本上可用```
+
+ - [flink-sql-connector-elasticsearch7_2.11-1.13.2.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.13.2/flink-sql-connector-elasticsearch7_2.11-1.13.2.jar)
+ - [flink-sql-connector-mongodb-cdc-2.2-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mongodb-cdc/2.2-SNAPSHOT/flink-sql-connector-postgres-cdc-2.2-SNAPSHOT.jar)
+
+4. 然后启动 Flink 集群，再启动 SQL CLI.
+
+```sql
+-- Flink SQL
+-- 设置间隔时间为3秒                       
+Flink SQL> SET execution.checkpointing.interval = 3s;
+
+-- 设置本地时区为 Asia/Shanghai
+Flink SQL> SET table.local-time-zone = Asia/Shanghai;
+
+Flink SQL> CREATE TABLE orders (
+   _id STRING,
+   order_id INT,
+   order_date TIMESTAMP_LTZ(3),
+   customer_id INT,
+   price DECIMAL(10, 5),
+   product ROW<name STRING, description STRING>,
+   order_status BOOLEAN,
+   PRIMARY KEY (_id) NOT ENFORCED
+ ) WITH (
+   'connector' = 'mongodb-cdc',
+   'hosts' = 'localhost:27017',
+   'username' = 'mongouser',
+   'password' = 'mongopw',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)